src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
-url=jdbc:sqlserver://t8serve.database.windows.net:1433;database=TateDB;user=tatettw@t8serve;password=TvTracker&;encrypt=true;trustServerCertificate=false;hostNameInCertificate=*.database.windows.net;loginTimeout=30;
-user=tatettw
-password=TvTracker&
+spring.jpa.hibernate.ddl-auto=validate
+spring.datasource.url=jdbc:mysql://localhost:3306/tvtracker
+spring.datasource.user=tatettw
+spring.datasource.password=TvTracker&
```

Comment:
Change from sqlserver to mysql

Reply:
It looks like they are already using a database... does switching to mysql reduce technical debt in any manner?